roles/postgres/pkg/tasks/update.yml

```diff
@@ -15,49 +15,83 @@
   vars:
     list_varname: _all_postgres_packages
 
-- set_fact:
-    _bdr_version: "{{ bdr_version }}"
-  when: "'bdr' in role"
+- name: Set target_bdr_version
+  set_fact:
+    target_bdr_version: "{{ bdr_version }}"
```

Comment:
Is bdr_version defined for non-bdr instances?

Reply:
BDR version is a cluster_var, I think we need it for the pgextended package change to determine if we need to switch to edbpge or not, that's why the bdr in role got removed and the following bdr package removal when clause got change to test for bdr in role first and stop the logic at this point to avoid PEM server to hit an undefined var.

Reply:
Sure, understood, but it won't be defined for M1?

Reply:
We could perhaps address that later as well when we do a proper PG upgrade for M1

Reply:
yes maybe some splitting will be needed to detach bdr related tasks from postgres.